android/KMEA/app/src/main/java/com/tavultesoft/kmea/util/FileUtils.java

```diff
@@ -210,9 +216,22 @@ public static String getFilename(String urlStr) {
     return filename;
   }
 
+  /**
+   * Utility if a given file is a font. We prefer TTF or OTF, but old Android devices use SVG or WOFF.
+   * Limitations: Include SVG only for Build.VERSION.SDK_INT == JELLY_BEAN_MR1 or JELLY_BEAN_MR2 for OSK
```

Comment:
I don't think we should limit this on the OS version. SVG fonts continue to work so an older package will work on a newer device, if we let it...